entrypoint

```diff
@@ -160,6 +160,16 @@ else
     # make the dump
     mysqldump -h $DBSERVER -u$DBUSER -p$DBPASS $DB_LIST | gzip > ${TMPDIR}/${TARGET}
 
+    # Execute additional scripts for post porcessing. For example, create a new
+    # backup file containing this db backup and a second tar file with the
+    # contents of a wordpress install.
+    if [ "$(ls -A /scripts.d/post-backup/)" ];
+    then
+      for i in $(ls /scripts.d/post-backup/*.sh);do
+          source $i
```

Comment:
> I would not know which ones should be exposed and which ones not.

That is precisely the issue. It becomes an undocumented API. Who knows what people will or will not assume about it? This will, in turn, tie down any future development because it will break something. Undocumented APIs always are a recipe for trouble (in the best of cases).

It also makes it easy to break things, since it is a single process rather than sourced in. 

There is a very good reason that every init system executes instead of includes.